Kwenta/kwenta
# Feat: new toggle for charts and history table

## Description
1. Update the chart and history toggle to the latest design, making them more consistent.
2. Adjust the position of the `HoursToggle`.

## Related issue
closes #2472 

## Motivation and Context


## How Has This Been Tested?
1. Open the market page, and click on `Price` and `Funding` to see the change in the chart.
2. Open the market page, and click on `Show` and `Hide` to see the change of the history table.

## Screenshots (if appropriate):
![image](https://github.com/Kwenta/kwenta/assets/4819006/c6729a85-9734-4e9d-89e3-4fd720325495)
